fail2ban/server/utils.py

```diff
@@ -116,7 +117,22 @@ def setFBlockMode(fhandle, value):
 		return flags
 
 	@staticmethod
-	def executeCmd(realCmd, timeout=60, shell=True, output=False, tout_kill_tree=True, success_codes=(0,)):
+	def buildShellCmd(realCmd, varsDict):
```

Comment:
as for order of items: no matter as long as variable statement (varsStat) and arguments list build in the same function (direct here in buildShellCmd).